src/Node/Node.ts

```diff
@@ -45,6 +45,9 @@ export default abstract class Node {
    */
   public originChain: string;
 
+  /** Path of bootnodes file. */
+  public bootNodesFile: string;
```

Comment:
bootNodes are supported only by Geth and not by Parity. Did we try to have this variable in `GethNode.ts` instead ?

Reply:
```suggestion
  public bootNodesFile?: string;
```

also as --bootnodes is an optional param